CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 
-## 2.2.2
+## 2.2.0
```

Comment:
issue: the version is 2.3.0 per your package json.

Reply:
updated